content/sse/index.textile

```diff
@@ -0,0 +1,119 @@
+- separator := (optional) a separator, to enable easy subscriptions to channels with commas in their name. For example, @?separator=|&channel=fo,o|ba,r@ will subcribe to the two channels @fo,o@ and @ba,r@.
+- v := (mandatory) the version of the api you are requesting. The current (and only) version of the API is 1.1, so only @v=1.1@ is accepted.
+- key := (optional) an Ably API key to use, if using basic auth.
+- accessToken := (optional) an Ably auth token to use, if using token auth.
+- lastEvent := (optional) an @id@ to resume from, see "connection state recovery":#recovery below.
+- enveloped := (optional, default true) if true, the @data@ from each event envelope for a @message@ event will be a "Message":/realtime/types#message object. If false, it will be the payload from the message directly. See "Envelope format":#envelope-format below.
+- heartbeats := (optional, default false) if @true@ will use an explicit heartbeat event rather than a newline as a keepalive packet.
+
+h5(#envelope-format). Envelope format
+
+Once a streaming response is established, every line (other than empty lines sent as keepalive packets) will be a simple JSON object of the following form:
+
+bc[json]. {
+  event: <string, the event type, such as "message", "presence", "error", or "heartbeat">,
+  data: <string, the message data, see below. Not present for some event types,
+         e.g. heartbeats>,
+  id: <string, the id to use to resume from this point, see 'connection state
+       recovery' below. Only present for message and presence events>
+}
+
+If @enveloped@ is true (the default), the @data@ will be a JSON-stringified object of a type determined by the event:
+- For a @message@ event it will be a "Message":/realtime/types#message object
+- For a @presence@ event it will be a "PresenceMessage":/realtime/types#presence-message object
+- For an @error@ event it will be an "ErrorInfo":/realtime/types#error-info object
+
+For a payload that is anything other than a string, as there is no client library to decode the payload of a @Message@ or @PresenceMessage@, you will have to decode it yourself. Objects and arrays will be json-encoded; binary payloads will be base64-encoded. The @encoding@ field of the @Message@ will specify what encoding has been done on it.
+
+If @enveloped@ is false, the @data@ will for a @message@ or @presence@ event be the @data@ payload from the @Message@ or @PresenceMessage@. Other events are unaffected.
+
+Non-string payloads will be encoded as before, but without enveloping you will not have the benefit of the @encoding@ field to tell you what encoding has been done.
+
+Note that failures on opening the connection (for example, invalid authentication details) may be sent as a non-streamed http response (with a response body of the form @{"error": <ErrorInfo>}@), not an @error@ event in a streamed response.
+
+h5. Example
+
+bc[sh]. curl "https://rest.ably.io/event-stream?key={{API KEY}}&channel=example&v=1.1"
```

Comment:
We could easily provide a trivial example in a JSBin and inline that subscribes via XHR, and resumes automatically setting `lastEvent`